synthesizer/event.h

```diff
@@ -22,8 +22,11 @@ class XmlWriter;
 
 enum class BeatType : char;
 
-// 4 is the default for the majority of synthesisers, aka VSTis
-const int PITCH_BEND_SENSITIVITY = 4;
+// 2 is the default for the majority of synthesisers, aka VSTis
```

Comment:
So, 4 or 2 is the default? :)

Reply:
Sorry for the late reply, 2 is default for DAWs but I found the 4 value through trial and error and changing the other pitch radius constant.

Reply:
So was the previous comment incorrect? If so, of course it is good that it gets corrected.

Reply:
@dmitrio95 Do you need that question answered to merge this pull request?